go-controller/pkg/node/healthcheck.go

```diff
@@ -217,16 +218,24 @@ func checkForStaleOVSInterfaces(stopChan chan struct{}, nodeName string, wf fact
 }
 
 type openflowManager struct {
-	gwBridge    string
-	physIntf    string
-	patchIntf   string
-	ofportPhys  string
-	ofportPatch string
+	gwBridge        string
+	physIntf        string
+	patchIntf       string
+	ofportPhys      string
+	ofportPatch     string
+	exGWGwBridge    string
+	exGWPhysIntf    string
+	exGWPatchIntf   string
+	exGWOfportPatch string
+	exGWOfportPhys  string
 	// flow cache, use map instead of array for readability when debugging
-	flowCache map[string][]string
-	flowMutex sync.Mutex
+	flowCache     map[string][]string
+	flowMutex     sync.Mutex
+	exGWFlowCache map[string][]string
+	exGWFlowMutex sync.Mutex
 	// channel to indicate we need to update flows immediately
-	flowChan chan struct{}
+	flowChan   chan struct{}
+	exGWBridge bool
```

Comment:
oh here it is...we need some comments or rework on this struct. Not sure you need a bool and a string to hold exGWBridge

Reply:
I reworked the struct. Not sure if it still works, but at least looks better :-)